os/board/rtl8721csm/src/component/common/mbed/targets/hal/rtl8721d/timer_api.c

```diff
@@ -79,8 +79,14 @@ void gtimer_init (gtimer_t *obj, uint32_t tid)
 	TIM_InitStruct.TIM_UpdateEvent = ENABLE; /* UEV enable */
 	TIM_InitStruct.TIM_UpdateSource = TIM_UpdateSource_Overflow;
 	TIM_InitStruct.TIM_ARRProtection = ENABLE;
-	
+#ifndef CONFIG_PLATFORM_TIZENRT_OS
 	RTIM_TimeBaseInit(TIM_x[tid], &TIM_InitStruct, TIMx_irq[tid], (IRQ_FUN) gtimer_timeout_handler, (u32)obj);
+#else
+	RTIM_TimeBaseInit(TIM_x[tid], &TIM_InitStruct, TIMx_irq[tid], NULL, NULL);

```

Comment:
Could you please add some comment regarding this?
If below is added to enable/disable interrupt, then amebad_timer_lowerhalf.c need to be fix.
(For example disable interrupt in setcallback if priv->callback is null)
I am right, it seems that start & stop also need to be fixed.

Reply:
This change is made in order to init the handler and enable the interrupt via Tizenrt_service.c instead of via rom code.
For the "disable interrupt in setcallback if priv->callback is null", I have added gtimer_deinit to disable the interrupt.
For "start & stop also need to be fixed", may I ask for the detail about it?

Reply:
Yes, gtimer_init is enough. It sets ISR & enable interrupt and gtimer_deinit disables timer, closes and clears interrupt.

Reply:
Got it but please add description to use it. If it is not useful, I mean gtimer_init & deinit is enough, then remove it. Build warning can be occured.

Reply:
Actually, the original 'RTIM_TimeBaseInit' includes ISR set and interrupt enable. But it was registered in the rom code. I made the change to prevent the registration in rom code via RTIM_TimeBaseInit. So that the interrupt and ISR can be set via Tizenrt_service.c instead.
I will add description for that, thanks!